subsys/emds/Kconfig

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig EMDS
+	bool
+	prompt "Emergency Data Storage"
+	default n
+	help
+	  Enable Emergency Data Storage.
+
+if EMDS
+
+config EMDS_SECTOR_SIZE_MULT
+	int "Sector size of the emergency data storage area"
+	default 1
+	help
+	  The sector size to use for the emergency data storage area as a
+	  multiple of FLASH_ERASE_BLOCK_SIZE.
+
+config EMDS_SECTOR_COUNT
+	int "Sector count of the emergency data storage area"
+	default 1
+	help
+	  Number of sectors used for the emergency data storage area
+
+choice
+	prompt "Initial log level limit"
+	default EMDS_LOG_LEVEL_WRN
```

Comment:
why are you not using the logging template here ?

For example like this:
https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/bluetooth/Kconfig.scan#L158-L160